review.php

```diff
@@ -0,0 +1,25 @@
+<?php
+session_start();
+
+require_once  'navbar.php';
+
+$db = new PDO('mysql:host=db; dbname=gameCollection', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
```

Comment:
can add all this into a function within a functions file, require it once and then call the function to avoid repeated code